### PR TITLE
Add alt tag generator example

### PR DESCRIPTION
**Pull request by Nutlope**
### Description

Added a alt tag generator that will generate relevant alt text for images using an ML model. First time contributing to this repo so please let me know if I missed anything or if there'd be a better folder for this!

I'm going to be adding this to templates marketplace after this is merged, and sharing it on Twitter.

### Type of Change

- [x] New Example
- [ ] Example updates (Bug fixes, new features, etc.)
- [ ] Other (changes to the codebase, but not to examples)

### New Example Checklist

- [ ] 🛫 `npm run new-example` was used to create the example
- [x] 📚 The template wasn't used but I carefuly read the [Adding a new example](https://github.com/vercel/examples#adding-a-new-example) steps and implemented them in the example
- [x] 📱 Is it responsive? Are mobile and tablets considered?

### CODE REVIEW

File: solutions/alt-tag-generator/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <h1 className={styles.h1}> Alt Image Generator</h1>
+      <p>
+        Send a request to `/api/generate` with `imageUrl` as a query parameter:
+      </p>
+      <a
+        className={styles.highlight}
+        href="/generate?imageUrl=https://dub.sh/confpic"
```
Comment (cond0r) [inline solutions/alt-tag-generator/pages/index.tsx:20]: This results in 404, it needs to be changed to /api/generate

Comment (Nutlope) [reply solutions/alt-tag-generator/pages/index.tsx:20]: Good catch, fixing now

File: solutions/alt-tag-generator/pages/api/generate.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    // Loop in 500ms intervals until the alt text is ready
+    let finalResponse = await fetch(endpointUrl, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: "Token " + process.env.REPLICATE_API_KEY,
+      },
+    });
+    let jsonFinalResponse = await finalResponse.json();
+    if (jsonFinalResponse.status === "succeeded") {
```
Comment (cond0r) [inline solutions/alt-tag-generator/pages/api/generate.ts:44]: Shouldn't check for a faild status and break? Or I was thinking about some maximum attempts, to not run this until termination.

**Review by cond0r — APPROVED** [review]

File: solutions/alt-tag-generator/pages/api/generate.ts (hunk 2 above)
Comment (cond0r) [reply solutions/alt-tag-generator/pages/api/generate.ts:44]: There is a type error after fail

**Review by cond0r — CHANGES_REQUESTED** [review]
There is a type error after the fail break; maybe set an error message as altText, or an empty string

**Review by cond0r — APPROVED** [review]

**Pull request merged**